jujugui/static/gui/src/app/components/deployment-flow/signup/signup.js

```diff
@@ -79,59 +80,53 @@ YUI.add('deployment-signup', function() {
+              <div className="six-col">
+                <h2>Install Juju to deploy locally</h2>
+                <p>
+                  Local deployment uses LXD containers, allowing you to
+                  recreate your production environment on your own machine.
+                  This minimises portability issues when deploying to a public
+                  cloud, OpenStack or bare metal.
+                </p>
+                <p>
+                  Continue to the&nbsp;
+                  <juju.components.GenericButton
+                    action={this._displayFlow}
+                    type="inline-neutral"
+                    title="deployment demo of Juju" />
+                </p>
+              </div>
             </div>
-            <div className="twelve-col">
-              <h2>Lorem ipsum dolor sit</h2>
-              <p className="six-col">
-                Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed
-                do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-              </p>
-              <ul>
-                <li className="six-col">
-                  Lorem ipsum
-                </li>
-                <li className="six-col last-col">
-                  Onsectetur adipisicing elit
-                </li>
-                <li className="six-col">
-                Sed do eiusmod
-                </li>
-                <li className="six-col last-col">
-                Tempor incididunt ut
-                </li>
-                <li className="six-col">Labore et dolore magna aliqua</li>
-              </ul>
-              <form className="twelve-col">
-                <juju.components.GenericInput
-                  label="Lorem ipsum dolor"
-                  required={true}
-                  ref="email"
-                  validate={[{
-                    regex: /\S+/,
-                    error: 'This field is required.'
-                  }]}
-                  value="" />
+            <div className="row">
+              <div className="twelve-col">
+                <h2>A new way to deploy</h2>
+                <p className="six-col">
+                  Coming soon: deploy from hosted Juju direct to public clouds.
+                  For early access to this feature, sign up for the beta.
+                </p>
+                <ul className="six-col">
+                  <li>
+                    Deploy to all major public clouds directly from your browser
+                  </li>
+                  <li>
+                    Hosted and managed juju controllers
+                  </li>
+                  <li>
+                    Identity management across all models
+                  </li>
+                  <li>
+                    Reusable shareable models with unlimited users
+                  </li>
+                </ul>
+                <div className="six-col">
                   <juju.components.GenericButton
                     action={this._handleSignup}
```

Comment:
This will have to be changed to an anchor tag with `target="_blank"`. The above `window.open` will be blocked by the browsers.